frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -6,54 +6,36 @@ import {
     PrimaryNav,
     Title,
 } from "@trussworks/react-uswds";
-import { NavLink } from "react-router-dom";
 import { NetworkErrorBoundary } from "rest-hooks";
 
 import { permissionCheck, PERMISSIONS } from "../../utils/PermissionsUtils";
 import { ReactComponent as RightLeftArrows } from "../../content/right-left-arrows.svg";
 import { useSessionContext } from "../../contexts/SessionContext";
 import { MemberType } from "../../hooks/UseOktaMemberships";
 import config from "../../config";
+import { USLink, USNavLink } from "../USLink";
 
 import { SignInOrUser } from "./SignInOrUser";
 import { AdminDropdown } from "./DropdownNav";
 
 const { IS_PREVIEW, CLIENT_ENV } = config;
 
 const ProductIA = () => (
-    <NavLink
-        to="/product"
-        key="product"
-        data-attribute="hidden"
-        hidden={true}
```

Comment:
Huh!  I'm guessing there was no need for this since it's getting removed now, but do you know what the intention of this prop was?  I guess it was just accompanying the `data-attribute` but not really doing anything 😬

Reply:
I, for the life of me, could not find a use. Nor was it actually _hidden_ in the UI. 🤷🏼 I figured it was copy/paste from the startup days.

Reply:
Yeah, I have no idea 😆 I know that this can be used for inputs for bot honeypots, but as far as I know `hidden` isn't a valid prop for NavLink so I have no idea why it was applied.  Either way, good to get rid of it!